front/pages/w/[wId]/builder/assistants/create.tsx

```diff
@@ -98,6 +98,29 @@ export default function CreateAssistant({
     });
   }, [assistantTemplates]);
 
+  const openTemplateModal = useCallback(
```

Comment:
`nit` `onTemplateModelOpen`

Reply:
If that's okay for you, I prefer "openTemplateModal" since this is what's doing the action of opening the modal.